low-level/framework/sspl_ll_d

```diff
@@ -717,21 +717,19 @@ def send_thread_controller_request(state):
         }
     thread_controller_queue.put((json.dumps(state_change_msg), None))
 
-def set_log_level(signal_number, frame):
-    """Handles SIGUSR1 for dynamic log level setting.
-       Key log_level should be registered in consul.
-       On SIGHUP signal reception, this function lookup
-       consul for latest value set with log_level key.
-    """
-    key='sspl/config/SYSTEM_INFORMATION/log_level'
-    logger.info("signal handler is called with {0}".format(signal_number))
-    log_level = store.get(key)
-    if log_level and isinstance(log_level, bytes):
-        log_level = log_level.decode('utf-8')
-    logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
-    logger.setLevel(log_level)
-    #DO NOT EDIT: Marker comment to dynamically add code to stop coverage, save and generate code coverage report
-
+# def set_log_level(signal_number, frame):
```

Comment:
Add a note here
\# Dynamic config change including log level set will ne enabled through ConfStor. 
\#On getting a trigger, this same function can be used back to set the required log level, excluding its interaction with Consul.

Reply:
added.